README.md

```diff
@@ -1,7 +1,21 @@
 # bird
 
-framework for web (it's my personal template for starting a new web project :D )
```

Comment:
dont remove this line, it's a description for bird project, move below line to how to use section